hass_nabucasa/client.py

```diff
@@ -75,7 +75,11 @@ async def logout_cleanups(self) -> None:
 
     @abstractmethod
     async def async_cloud_connect_update(self, connect: bool) -> None:
-        """Process cloud remote message to client."""
+        """Process cloud remote message to client.
+
+        If it's not allowed to remotely enable remote control, the implementation
+        should raise HandlerError("remote_activation_not_allowed")
```

Comment:
We could also add a specific subclass of `HandlerError`, `RemoteActivationNotAllowed`

Reply:
Yes, that's better, I think.